test/parallel/test-async-hooks-execution-async-resource-await.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const common = require('../common');
+const sleep = require('util').promisify(setTimeout);
+const assert = require('assert');
+const { executionAsyncResource, createHook } = require('async_hooks');
+const { createServer, get } = require('http');
+const sym = Symbol('cls');
+const id = Symbol('id');
+
+// Tests continuation local storage with the currentResource API
+// through an async function
+
+assert.ok(executionAsyncResource());
+
+createHook({
+  init(asyncId, type, triggerAsyncId, resource) {
+    const cr = executionAsyncResource();
+    resource[id] = asyncId;
```

Comment:
Why is this done? it's never read later.

Reply:
You're right. Not needed. Fixed.